gourmet/Undo.py

```diff
@@ -267,15 +268,16 @@ def _set_text (self,txt):
     def set_text (self,txt,cursor_index): raise NotImplementedError
 
 class UndoableEntry (UndoableTextContainer):
-    def __init__ (self, entry,history):
+    def __init__(self, entry: Gtk.Widget, history):
         self.entry = entry
-        self.get_text = self.entry.get_text
+        if isinstance(entry, Gtk.ComboBoxText):
+            self.get_text = entry.get_active_text()
+        elif isinstance(entry, Gtk.Entry):
+            self.get_text = entry.get_text()
         UndoableTextContainer.__init__(self,self.entry,history)
 
     def setup_widgets (self):
-        self.entry.connect('changed',
-                           self.change_event_cb
-                           )
+        self.entry.connect("changed", self.change_event_cb)
```

Comment:
This is just a whitespace change but looks like we might have had to touch it up. Maybe you can remove that change and force push that commit so this change doesn't show up

Reply:
You're absolutely right.